.travis.yml

```diff
@@ -26,6 +26,6 @@ jdk:
 before_script:
   - unset _JAVA_OPTIONS
 env:
-  - BUILD_ARGS="clean verify javadoc:jar"
+  - BUILD_ARGS="-B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn clean verify javadoc:jar"
```

Comment:
I would put these options in the script section instead. In case you want to have a matrix build with other maven goals, you don't want to copy this to each line.

Reply:
Can you provide an example?

Reply:
I made a a change.

Reply:
Basically, instead of:

```yaml
env:
  - BUILD_ARGS="-B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn clean verify javadoc:jar"
script:
  - mvn $BUILD_ARGS
```

You could do:

```yaml
env:
  - BUILD_ARGS="clean verify javadoc:jar"
script:
  - mvn $BUILD_ARGS -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Both have the same result now, but in future, it's easier to add a second execution by adding a new BUILD_ARGS line without having to move around the suppression stuff.